packages/noya-renderer/src/components/EditablePath.tsx

```diff
@@ -104,6 +148,36 @@ export default function EditablePath({
           />
         );
       })}
+
+      {parsedCurvePoints.map((points, index) => {
+        if (points.curveMode === 1) {
+          return null;
+        }
+        return (
+          <ControlPathPoint
+            key={index}
+            point={points.point}
+            controlPoint={points.curveFrom}
+            fill={fill}
+            stroke={stroke}
+          />
+        );
+      })}
+
+      {parsedCurvePoints.map((points, index) => {
```

Comment:
These lines currently render over the purple "point" oval. We should render them beneath instead, by moving them up above `{points.map(...)}`

![Screen Shot 2021-06-09 at 8 03 10 PM](https://user-images.githubusercontent.com/1198882/121458151-b8881680-c95d-11eb-8216-85080bd029ae.png)

Reply:
Done.